### DISCUSSION TITLE
BarChart: Allow stacking for separate time series

### DISCUSSION THREAD

**Discussion by thoro** [grafana/grafana#59634] [Dashboards, Panels & Visualizations]
Hello,

I'm visualizing power consumption, solar production, grid feed-in and would like to stack only some bars and not all of them.

The BarChart is missing the Override property for the stack option.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
